### PR TITLE
Fix WP nonce verification issue   #130

### PR DESCRIPTION
**Pull request by hasnain37**

### CODE REVIEW

File: wp-frontend-profile.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * Plugin Name: WP Frontend Profile
  * Plugin URI: https://wordpress.org/plugins/wp-front-end-profile/
  * Description: This plugin allows users to easily edit their profile information on the frontend rather than having to go into the dashboard to make changes to password, email address and other user meta data.
- * Version:     1.3.2
+ * Version:     1.3.3
```
Comment (musamamasood) [inline wp-frontend-profile.php:6]: Require to change version for **WPFEP_VERSION**

Comment (hasnain37) [reply wp-frontend-profile.php:6]: @musamamasood fixed.

File: inc/class-wpfep-login.php [hunk 2]
```diff
@@ -426,69 +433,69 @@ public function process_reset_password()
...
                             return;
                         }
-                    }
 
-                    $errors = new WP_Error();
+                        if (! $this->login_errors) {
+                            $this->reset_password($user, $_POST['pass1']);
 
-                    do_action('validate_password_reset', $errors, $user);
+                            do_action('wpfep_customer_reset_password', $user);
```
Comment (musamamasood) [inline inc/class-wpfep-login.php:489]: Why $errors remove from **do_action**

**Review by musamamasood — APPROVED** [review]
Approve

**Pull request merged**